extensions/wikidata/src/org/openrefine/wikidata/schema/entityvalues/SuggestedItemIdValue.java

```diff
@@ -31,6 +31,11 @@ public SuggestedItemIdValue(String id, String siteIRI, String label) {
         super(id, siteIRI, label);
     }
 
+	@Override
+	public boolean isPlaceholder() {
+		return true;
```

Comment:
For items coming from a suggest service, we know they always exist, so this should be `return false` (and similarly for other `Suggested*IdValue`).